src/javascript/app/common/guide.js

```diff
@@ -38,18 +39,14 @@ const Guide = (() => {
         if ($(opt.guideBtnID).length === 0 || opt.script.length === 0) {
             return;
         }
-
+       
         if (isDisabled()) {
             $(opt.guideBtnID).remove();
             return;
         }
-
         makeButton();
     };
-
-    /*
-     *  do not show the guide button if its close (X) has been clicked before
-     */
```

Comment:
I think we don't need to remove this comment now

Reply:
Returned.